tests/testthat/test_Widget_BarChart.R

```diff
@@ -40,3 +40,60 @@ test_that("Widget_BarChart processes dfGRoups correctly", {
   reportingGroups_json <- jsonlite::toJSON(reportingGroups, na = "string")
   expect_equal(widget$x$dfGroups, reportingGroups_json)
 })
+
+test_that("Widget_BarChart assertions works", {
```

Comment:
@samussiah did you want the unit test for the respective argument in that same `test_that` chunk? For now I just slotted them all as an "assertions" chunk at the end of each file

Reply:
Makes sense to me!